core/src/main/java/bio/terra/pearl/core/service/rule/EnrolleeProfileBundle.java

```diff
@@ -4,9 +4,12 @@
 import bio.terra.pearl.core.model.participant.Profile;
 import lombok.AllArgsConstructor;
 import lombok.Getter;
+import lombok.experimental.SuperBuilder;
 
-@Getter @AllArgsConstructor
-public class EnrolleeRuleData {
+@Getter
+@AllArgsConstructor
+@SuperBuilder
+public class EnrolleeProfileBundle {
```

Comment:
I am not sure if this is the best name for this class. It was originally `EnrolleeRuleData`. I wanted to delete it in favor of `EnrolleeSearchContext`, but I noticed it was being used in a bunch of unrelated places to rules.  I thought making it a `Bundle` with a corresponding `EnrolleeBundleService` could be more generic, but I am not 100% sure about it.